.gitignore

```diff
@@ -92,21 +107,37 @@
 /test/offer_tests/offer_test_big_sd_msg_slave.json
 /test/subscribe_notify_tests/subscribe_notify_test_diff_client_ids_diff_ports_autoconfig_master.json
 /test/subscribe_notify_tests/subscribe_notify_test_diff_client_ids_diff_ports_autoconfig_slave.json
-test/security_tests/security_test_config_client_external_allow.json
-test/security_tests/security_test_config_client_external_deny.json
-test/security_tests/security_test_config_service_external_allow.json
-test/security_tests/security_test_config_service_external_deny.json
-test/security_tests/security_test_local_config.json
+/test/network_tests/security_tests/security_test_config_client_external_allow.json
+/test/network_tests/security_tests/security_test_config_client_external_deny.json
+/test/network_tests/security_tests/security_test_config_service_external_allow.json
+/test/network_tests/security_tests/security_test_config_service_external_deny.json
+/test/network_tests/security_tests/security_test_local_config.json
 /test/pending_subscription_tests/pending_subscription_test_master.json
 /test/pending_subscription_tests/pending_subscription_test_master_starter.sh
 /test/malicious_data_tests/malicious_data_test_master.json
 /test/malicious_data_tests/malicious_data_test_master_starter.sh
-/test/debug_diag_job_plugin_tests/debug_diag_job_plugin_test_master.json
-/test/debug_diag_job_plugin_tests/debug_diag_job_plugin_test_slave.json
-/test/security_config_plugin_tests/security_config_plugin_test_local.json
+/test/e2e_tests/e2e_profile_04_test_client_external.json
+/test/e2e_tests/e2e_profile_04_test_service_external.json
 /test/e2e_tests/e2e_test_client_external.json
 /test/e2e_tests/e2e_test_service_external.json
 /test/event_tests/event_test_master.json
 /test/event_tests/event_test_slave_tcp.json
 /test/event_tests/event_test_slave_udp.json
-!build_qnx/*
```

Comment:
I'm not sure if this is meant to be removed. It was explicitly added with the QNX build support.

Reply:
you're correct, there seems to be an error while updating the .gitignore file, I will address it.